joschahenningsen/TUM-Live
# Switch to go-pts package

### Motivation and Context
The real-time code was excluded into its own package.
This package keeps a small scope and just focuses on well-tested real-time channel management.

### Description
Realtime code was removed and replaced by a go import of go-pts.

### Steps for Testing
To test the changes, you can verify if realtime actions still work.

Prerequisites:
- 1 Lecturer
- 1 Students
- 1 Livestream

1. Log in
2. Navigate to a Livestream
3. Type chat messages
4. Check if they are well received by the other account
